swap/swap.go

```diff
@@ -147,68 +141,37 @@ func (s *Swap) DeploySuccess() string {
 func (s *Swap) Add(amount int64, peer *protocols.Peer) (err error) {
 	s.accountingLock.Lock()
```

Comment:
we are probably going to end up removing this mutex completely if we want to lock only per peer (and not generally)

Reply:
We do access some of the shared swap fields but all of them read-only (and they are not maps and never change).